activestorage/test/models/variant_with_record_test.rb

```diff
@@ -105,8 +104,9 @@ class ActiveStorage::VariantWithRecordTest < ActiveSupport::TestCase
     users.reset
 
     assert_no_difference -> { ActiveStorage::VariantRecord.count } do
-      assert_queries_count(6) do
-        # 6 queries:
+      assert_queries_count(7) do
+        # 7 queries:
```

Comment:
Why this new query?

Reply:
This was because I had turned off eager load, but going to turn it back on to reduce the diff size.

https://github.com/rails/rails/blob/0add5dba834f2f1b84fcf1bd1b758545b325fb73/activestorage/test/dummy/config/environments/test.rb#L18